### DISCUSSION TITLE
urls with mvk formate are failing

### DISCUSSION THREAD

**Discussion by ashish2442** [kingslay/KSPlayer#830] [General]
Hi, can anyone please confirm if the MKV format is supported by this library? Where can I find a list of all supported formats? 
I see FFmpegKit and assumed it might support mvk. Any thing I am missing or need to enable? I would really appreciate any please. Thanks.

I am using KSVideoPlayerView in swiftUI iOS. 
if let url = viewModel.streamURL {
                KSVideoPlayerView(url: url, options: KSOptions())
            }
I am trying to play MKV URLs, but I am encountering the following error:

Warning KSPlayer: KSPlayerLayer.swift:168 state | playerStateDidChange - error
Warning KSPlayer: KSPlayerLayer.swift:451 finish(player:error:) | Error Domain=AVFoundationErrorDomain Code=-11828 "Cannot Open" 
UserInfo={NSLocalizedFailureReason=This media format is not supported., 
NSLocalizedDescription=Cannot Open, 
NSUnderlyingError=0x600001052610 {Error Domain=NSOSStatusErrorDomain Code=-12847 "(null)"}}

**Comment by kingslay** [comment]
你更新下最新的代码。
